src/components/radios/_macro.njk

```diff
@@ -18,8 +18,8 @@
                 {% if params.or is defined and params.or and loop.revindex0 == 0 %}
                     <span class="radios__label u-mt-s u-fs-r--b" aria-hidden="true">{{ params.or }} </span>
                 {% endif %}
-                <p class="radios__item">
-                    <span class="radio">
+                <p class="radios__item{{ " u-mb-s" if params.borderless else "" }}">
```

Comment:
Might be better to handle the margins in the css as the bordered variant has the margin on the `:last-child` removed, which should be the same here for this variant.